docs/deprecated.md

```diff
@@ -39,10 +40,10 @@ information in the case you want to use old versions:
 The following table provides information about the last lightweightM2M-iotagent version supporting currently removed
 features:
 
-| **Removed feature**   | **Last lightweightM2M-iotagent version supporting feature** | **That version release date** |
-| --------------------- | ----------------------------------------------------------- | ----------------------------- |
-| NGSIv1 API            | Not yet defined                                             | Not yet defined               |
-| Support to Node.js v4 | 0.4.0                                                       | November 28th, 2017           |
-| Support to Node.js v6 | May 22nd, 2019              | 1.1.0               |
-| Support to Node.js v8 | April 8th, 2020             | 1.4.0               |
-| Support to Node.js v6 | March 3rd, 2021             | 1.5.0               |
+| **Removed feature**    | **Last lightweightM2M-iotagent version supporting feature** | **That version release date** |
```

Comment:
It seems the content in second and third columns should be swapped. Could you do it taking advantage of this PR, pls?

Reply:
Fixed d6c8b70